angular-ngrx-scss/src/app/home/profile/profile-nav/profile-nav.component.html

```diff
@@ -0,0 +1,10 @@
+<nav>
+  <ul>
+    <li class="active-tab">
+      <a routerLink="/{{ (profile$ | async)?.user?.username }}">
```

Comment:
This is for the repositories for the user you are viewing so that's why the username is passed there. It's following GitHub's path conventions.

Reply:
So I think for now, we can leave this and look again when the profile folder is moved up a level. That might still work as expected and I do like keeping to GitHub's arrangement as much as possible, but since for our showcases the profile view is the only way to view a user's repos, it may not need the username. Can revisit this though, I could be wrong!